lib/floki/selector.ex

```diff
@@ -269,6 +273,7 @@ defmodule Floki.Selector do
 
   defp attributes(%HTMLNode{type: :pi}), do: []
   defp attributes(%HTMLNode{attributes: attributes}), do: attributes
+  defp attributes({_type, attributes, _children}), do: attributes
```

Comment:
I have the impression that if you move this match to the top of the other clauses, it can be slightly faster (because it's a simpler data structure).

Reply:
Done 5b04a3aebada2c2e0056b56bf176459e8715c616, benchmark values were pretty close, so hard to know if this actually improved the performance or not

Reply:
Nice! Thanks!